release-notes/opensearch-release-notes-2.19.1.md

```diff
@@ -0,0 +1,159 @@
+# OpenSearch and OpenSearch Dashboards 2.19.1 Release Notes
+
+
```

Comment:
Need to update core links and more.
See previous patch release notes in this repo.

Thanks.

Reply:
Sure, Let me update manually.